geometry_msgs/Point pose
geometry_msgs/Vector3 linear_vel
geometry_msgs/Vector3 angular_vel
